src/HttpClient/Response/HttpResponse.php

```diff
@@ -28,12 +29,15 @@ class HttpResponse implements ResponseInterface
      */
     private $document;
 
+    /** @var PsrResponseInterface */
+    protected $psrResponse;
+
     /**
      * @param int $status
      * @param array $headers
      * @param DocumentInterface|null $document
      */
-    public function __construct(int $status, array $headers, ?DocumentInterface $document)
+    public function __construct(int $status, array $headers, ?DocumentInterface $document, ?PsrResponseInterface $psrResponse = null)
```

Comment:
Missing documentation above. You can also choose to remove them completely.

Reply:
Removed the whole doc block